### PR TITLE
CLI: tweaks command defaults; add `version` command; handle invalid commands

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: bin/cli.js [hunk 1]
```diff
@@ -222,13 +249,16 @@ function packageApp() {
   });
 }
 
+function displayVersion() {
+  console.log(packageJson.version);
```
Comment (mykmelez) [inline bin/cli.js]: It'd be nice if `--version` also displayed the version of the runtime, but we can do this in a followup.

File: bin/postinstall.js [hunk 2]
```diff
@@ -235,7 +235,7 @@ new Promise((resolve, reject) => {
 .then(() => {
   // Copy and configure the stub executable.
 
-  switch(process.platform) {
+  switch (process.platform) {
```
Comment (mykmelez) [inline bin/postinstall.js]: It looks like we can enforce this space with the `keyword-spacing: ["error"]` eslint rule.

Comment (mykmelez) [inline bin/postinstall.js]: (I'm adding this over in #36.)

File: bin/cli.js (hunk 1 above)
Comment (mykmelez) [inline bin/cli.js]: (I filed this as #37.)

Comment (cvan) [inline bin/cli.js]: good idea - thanks!

File: bin/postinstall.js (hunk 2 above)
Comment (cvan) [inline bin/postinstall.js]: nice, thanks

**Pull request merged**